utils/ROI_Template.py

```diff
@@ -256,56 +397,63 @@ def roi_from_dict(self, **kwargs):
                 log.error(f"Mandatory key {mk} not present in kwargs {kwargs.keys()}")
                 raise Exception("Missing Mandatory Key")
 
-        self.handle = Handle(**kwargs.pop("Handle", {}))
-        self.seriesInstanceUid = kwargs.pop("SeriesInstanceUID")
-        self.sopInstanceUid = kwargs.pop("SOPInstanceUID")
-        self.studyInstanceUid = kwargs.pop("StudyInstanceUID")
-        self.patientId = kwargs.pop("patientId")
+        self.handle = Handle(**kwargs.pop(HANDLE_KWD, {}))
```

Comment:
I'll definitely use data class but I still want to use my own to_dict method BECAUSE given the volatility of the ohifViewer metadata, I may have to end up changing the name of a key in the future and I'd rather only have to make a change in one place.  For example, "cachedStats" was once "meanStdDev" or something.   so rather than change "cachedStats" to "meanStdDev" throughout the code, I just need to change my "to_dict" key for it.

Reply:
Wouldn't you only need to change it in one place in the definition of the class?

Reply:
if I'm understanding this correctly, a dataclass with a property `object.property = value` would automatically generate a dictionary `{property: value}`.  I want to be able to change the string value of `property`  in the output dictionary without having to go change every other place in the code that I call `object.property`.